src/stream.rs

```diff
@@ -51,6 +51,20 @@ impl OutputStream {
         Ok((out, handle))
     }
 
+    pub fn try_from_config(
+        device: &cpal::Device,
+        config: &StreamConfig,
+        sample_format: &SampleFormat,
+    ) -> Result<(Self, OutputStreamHandle), StreamError> {
+        let (mixer, _stream) = device.try_new_output_stream(&config, &sample_format)?;
+        _stream.play()?;
```

Comment:
What a wonderful pull request!!!
I had the opposite of your case, a problem with regeneration due to lack of buffer, but thanks to you I was able to solve it.
One thing I noticed is that this `_stream.play()? ` is necessary?
```suggestion
```
I removed it and tried it ([in this fork](https://github.com/pulsuite/rodio/tree/configurable-buffer-size-without-play)), but it works without any problems and the time it takes has become shorter.